tests/systems/camera.test.js

```diff
@@ -147,7 +147,7 @@ suite('camera system', function () {
           assert.ok(cameraEl.getAttribute('camera').active);
           assert.ok(cameraSystem.setActiveCamera.calledOnce);
           assert.equal(cameraSystem.activeCameraEl, cameraEl);
-          cameraSystem.setActiveCamera.reset();
+          cameraSystem.setActiveCamera.resetHistory();
```

Comment:
The sinon api changed. I already did a similar change in other files when I did #5091